examples/akka-persistence-app/build.sbt

```diff
@@ -25,7 +25,7 @@ val AkkaHttpVersion = "10.2.9"
 val AkkaManagementVersion = "1.1.3"
 val AkkaPersistenceJdbcVersion = "5.0.4"
 val AkkaProjectionVersion = "1.2.4"
-val ScalikeJdbcVersion = "3.5.0"
+val ScalikeJdbcVersion = "4.0.0"
```

Comment:
Hmmm maybe as suggested in PR description, leave the dependency at 3.5.0 plus add `updates.ignore = [ { groupId = "org.scalikejdbc" } ]` into steward config? With a detailed comment on what specifically fails